cloud-aws-common/src/test/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsCredentialVerifierTest.java

```diff
@@ -102,7 +102,7 @@ public void verifyCredentialAndThrowFailExceptionTest() throws IOException {
             assertThat(e.getMessage(), not(CoreMatchers.containsString("accepted_action")));
         }
         List<SimulatePrincipalPolicyRequest> allSimulatePrincipalPolicyRequest = requestArgumentCaptor.getAllValues();
-        int simulateRequestNumber = 4;
+        int simulateRequestNumber = 2;
```

Comment:
why only 2?

Reply:
@doktoric 
The key difference here is that in the old way we had 2 other Sids that has a resource other than '*': 

- The "LogsLimited" with the actions of ["logs:CreateLogStream", "logs:DescribeLogStreams", "logs:PutLogEvents"] for the resource "arn:aws:logs:\*:\*:log-group:/aws/eks/\*:\*" 
- And the "PerformanceInsightsLimited" for action "pi:\*" for the resource "arn:aws:pi:\*:\*:metrics/rds/\*"

The lack of these two entries make the test output different.